common/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 message("${BoldWhite}Preparing ${BoldGreen}common${BoldWhite} for the ${BoldGreen}${TARGET_PLATFORM}${BoldWhite} platform${ColourReset}")
 
 # Default values
-set(BEEROCKS_PLATFORM_INIT "false")
+set(BEEROCKS_PLATFORM_INIT "false" CACHE STRING "Configure minimal set of dummy WLAN ifaces and bridge")
```

Comment:
This variable is only used in the prplmesh_utils.sh script.
There, it can be controlled using the "--platform-init" option. Why would we additionally need to set the default to true?
Also, with this change and if BEEROCKS_PLATFORM_INIT is set to true, the "--platform-init" option from the script becomes unusable, which can be confusing.

Reply:
I also think it should be removed entirely, but as Ivan wrote somewhere else: that's not the purpose of this PR. As it is now, the PR nicely converts all existing options that should be configurable to cache variables, so I'm happy with it.